litmus-portal/frontend/src/components/DeveloperGuide/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Typography, IconButton } from '@material-ui/core';
+import * as React from 'react';
+import CloseIcon from '@material-ui/icons/Close';
+import useStyles from './styles';
+
+interface DeveloperGuideProps {
+  expAvailable: boolean;
+  header: string;
+  description: string;
+}
+
+const DeveloperGuide: React.FC<DeveloperGuideProps> = ({
+  expAvailable,
+  header,
+  description,
+}) => {
+  const [display, setDisplay] = React.useState(true);
+
+  const handleClose = () => {
+    setDisplay(false);
+  };
+  const docs = 'https://docs.litmuschaos.io/docs/getstarted/';
+
+  const classes = useStyles();
+  return (
+    <>
+      {display ? (
+        <div className={classes.root}>
+          <div style={{ display: 'flex', flexDirection: 'row' }}>
```

Comment:
Please consider adding these to the 'styles.ts' file. Also the default flexDirection is 'row' itself, it may be possible to not include that depending on how you configured the parent root.